app/src/main/java/org/thunderdog/challegram/data/TGStickerSetInfo.java

```diff
@@ -75,7 +80,8 @@ public TGStickerSetInfo (Tdlib tdlib, TdApi.Sticker[] stickers, boolean areFavor
   public TGStickerSetInfo (Tdlib tdlib, @NonNull TdApi.StickerSetInfo info) {
     this.tdlib = tdlib;
     this.info = info;
-    if (info.thumbnail != null) {
+    boolean iDontKnowHowFixThat = StringUtils.equalsOrBothEmpty(info.title, "Status Pack");
```

Comment:
What issue this is referring to? At least `setId` must be used, as there're pretty many sticker sets with `Status Pack` in its title.